### DISCUSSION TITLE
Manual switch activations not visible in zwavejs2mqtt

### DISCUSSION THREAD

**Discussion by NeilDarach** [zwave-js/zwave-js-ui#1402] [Q&A]
I've just installed zwavejs2mqtt as a docker image running on a Raspberry Pi 4 to control an existing network.
Host OS Alpine Linux 3.13, docker image zwavejs/zwavejs2mqtt:latest, zwavejs2mqtt 5.1.0, zwavejs 7.9.0.
ZWave controller is a Z-Wave.Me Razberry Controller 2016 ZWave+

In one room I have a TKB Home TZ66 dual paddle wall switch, the wired paddle controlling the main switched light and the second paddle controlling a number of side lights (TZ68 wall plugs and others).

The light is configured to notify the controller (Node 18) for all groups (group 1 for the main light, group 2, associated with the controller and 4 other nodes)

Lights can be operated from the switch, and from the zwavejs2mqtt gui, but when operated from the wall switch nothing shows up in the logs, and state isn't updated.

zwavejs.log & zwavejs2mqttl.log are tracking at log level 'Silly'

When operating from the GUI, I do see log entries

[26-134-0-libraryType] Library type  6
[26-134-0-protocolVersion] Z-Wave protocol version  2.97
[26-134-0-firmwareVersions] Z-Wave chip firmware versions   ["1.3"]

Another switch with the same device id (280-4128-258 (0x0118-0x0102-0x1020)) and version information does generate entries in the zwave log when switched manually.

Possibly related, though I don't think so, the controller is stuck in NodeInfo in the interview process and has been for days.

Is there anything I can change to get notifications to show up when the switch is operated manually?

Thanks

[mainLightOff-zwavejs2mqtt.log](https://github.com/zwave-js/zwavejs2mqtt/files/6759783/mainLightOff-zwavejs2mqtt.log)
[sideLightOff-zwavejs2mqtt.log](https://github.com/zwave-js/zwavejs2mqtt/files/6759784/sideLightOff-zwavejs2mqtt.log)
[sideLightOff-zwavejs.log](https://github.com/zwave-js/zwavejs2mqtt/files/6759785/sideLightOff-zwavejs.log)
[mainLightOff-zwavejs.log](https://github.com/zwave-js/zwavejs2mqtt/files/6759786/mainLightOff-zwavejs.log)

**Comment by robertsLando** [comment]
@AlCalzone

**Comment by AlCalzone** [comment]
> the controller is stuck in NodeInfo in the interview process

That's why you're not seeing any notifications. The node isn't determined to be `ready`, so zwavejs2mqtt doesn't show any info.

Re-interview the node and post a driver log of that.

**Reply by NeilDarach** [reply]
Clicking Node / Advanced / Interview generates a record of the button push in zwavejs2mqtt.log but nothing at all in zwavejs_2021-07-06.log.


2021-07-06   19:29:54.765 DEBUG SOCKET: Event ZWAVE_API emitted to T_FgBPsxo-3sHYhFAAAL
--
2021-07-06 19:29:54.769 INFO ZWAVE:   Calling api refreshInfo with args: [ 18, [length]: 1 ]
2021-07-06 19:29:54.773 INFO ZWAVE:   Success zwave api call refreshInfo undefined

**Reply by AlCalzone** [reply]
@robertsLando ?

**Reply by NeilDarach** [reply]
I tried adding a security key, though I've never needed one before, and restarted the docker container to make sure everything was applied.  No change on the interview status from that, but I did see the following in the log during a restart.

2021-07-07T21:17:25.670Z DRIVER   driver ready
2021-07-07T21:17:25.974Z CNTRLR   [Node 018] Beginning interview - last completed stage: NodeInfo
2021-07-07T21:17:25.976Z CNTRLR   [Node 018] not pinging the controller
2021-07-07T21:17:26.091Z CNTRLR   [Node 018] Interviewing Multi Channel...
2021-07-07T21:17:26.099Z CNTRLR .. [Node 018] Querying endpoint count for CommandClass Application Status...
2021-07-07T21:17:26.103Z CNTRLR   [Node 018] Error during node interview: This virtual node does not support the
                                   command GetV1!

**Reply by AlCalzone** [reply]
Obviously there is something in the driver log now. Please do a re-interview and post a complete log like I asked you to.

**Reply by robertsLando** [reply]
Plese be sure the loglevel is set to silly

**Reply by NeilDarach** [reply]
Full logs from a docker restart.  Reinterview controller and a refresh of one of the other nodes.
[zwavejs_2021-07-08.log](https://github.com/zwave-js/zwavejs2mqtt/files/6786267/zwavejs_2021-07-08.log)
[zwavejs2mqtt.log](https://github.com/zwave-js/zwavejs2mqtt/files/6786270/zwavejs2mqtt.log)


Deleting the json, metadata.json1 & values.json1 cache files while the container was stopped allowed the controller interview to complete.  But there are still no entries in the log when the manual switch is activated.

**Reply by AlCalzone** [reply]
I'm not seeing a re-interview of node 18.

**Reply by NeilDarach** [reply]
As I said, there's nothing in the logs for the failed interview.  You can see in the logs attached to the previous message a request for interview through the GUI in zwave2mqtt.log at 17:26:34, but there's nothing in zwavejs_2021-07-08.log.

Deleting the cache files and restarting allowed a startup interview to complete (17:40:33 in the logs attached to this message).

[zwavejs_2021-07-08.log](https://github.com/zwave-js/zwavejs2mqtt/files/6794025/zwavejs_2021-07-08.log)
[zwavejs2mqtt.log](https://github.com/zwave-js/zwavejs2mqtt/files/6794027/zwavejs2mqtt.log)


The controller interview has completed, but it hasn't affected the initial problem, that manually switching a set of lights is obviously sending out zwave instructions (because the lights turn on and off) but not showing anything at all in the logs, literally 0 bytes written to either log.

Groups on the wall switch seem to be set reasonably.

0       Group 1         Controller (Study)      None
0       Group 2         Bookcase Light (Living Room)    None
0       Group 2         Left Lamp (Living Room)         None
0       Group 2         Right Lamp (Living Room)        None
0       Group 2         Bookshelf Lamps (Living Room)   None
0       Group 2         Controller (Study)      None
0       Group 3         Controller (Study)      None
0       Group 4         Controller (Study)      None

**Reply by AlCalzone** [reply]
Ohhh my bad - I didn't realize that Node 18 is the controller. A controller interview does nothing besides what's happening on each startup anyways. I need an interview of the switch (Node 26?)
